shared/locales/media/media.en-US.properties

```diff
@@ -0,0 +1,3 @@
+playbackPlay.ariaLabel=Play
```

Comment:
I did not know that you could do this with localization. Nice feature!

Is it normal to leave 'playbackPlay' completely undefined in this case, or would it be better to explicitly set it to the empty string?  I assume that the way you have it is best, but just wanted to check since I have not seen this done before.